### DISCUSSION TITLE
Unable to overrule $persistentMiddleware

### DISCUSSION THREAD

**Discussion by francoism90** [livewire/livewire#6316] [1. Help]
It is not possible to overrule the `$persistentMiddleware` in `src/Mechanisms/PersistentMiddleware/PersistentMiddleware.php`.

This causes the following issues:
- Sometimes you need to define middlewares in a particular order
- Sometimes the class doesn't exists (anymore) in `\App\Http\..`.
Laravel can be customized, sometimes one uses a different namespace, like `\Foundation`.
- Using the config helper, also allows caching the middleware values.

**Comment by inmanturbo** [comment]
you mean in v3? Wasn't this possible in v2? I worked on an app that used middleware to set the database connection for a tenant, and I remember having to override the persistent middleware to get livewire working properly.

**Reply by francoism90** [reply]
@inmanturbo You can actually do this in a service provider instead of a config file. See my closed PR for details.
